packages/features/bookings/lib/handleNewBooking/scheduleNoShowTriggers.ts

```diff
@@ -0,0 +1,122 @@
+import dayjs from "@calcom/dayjs";
+import getWebhooks from "@calcom/features/webhooks/lib/getWebhooks";
+import prisma from "@calcom/prisma";
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
+
+type ScheduleNoShowTriggersArgs = {
+  booking: {
+    startTime: Date;
+    id: number;
+  };
+  triggerForUser: number | true | null;
+  organizerUser: { id: number };
+  eventTypeId: number;
+  teamId?: number | null;
+  orgId?: number | null;
+};
+
+export const scheduleNoShowTriggers = async (args: ScheduleNoShowTriggersArgs) => {
+  const { booking, triggerForUser, organizerUser, eventTypeId, teamId, orgId } = args;
+  // Add task for automatic no show in cal video
+  const noShowPromises: Promise<any>[] = [];
+
+  const subscriberHostsNoShowStarted = {
+    userId: triggerForUser ? organizerUser.id : null,
+    eventTypeId,
+    triggerEvent: WebhookTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW,
+    teamId,
+    orgId,
+  };
+
+  const subscribersHostsNoShowStarted = await getWebhooks(subscriberHostsNoShowStarted);
```

Comment:
Nit: Extracting this to a constant implies that this is being used else which it doesn't.

```suggestion
  const subscribersHostsNoShowStarted = await getWebhooks({
    userId: triggerForUser ? organizerUser.id : null,
    eventTypeId,
    triggerEvent: WebhookTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW,
    teamId,
    orgId,
  });
```